schema/bom-1.6.schema.json

```diff
@@ -409,10 +409,11 @@
             "firmware",
             "file",
             "machine-learning-model",
-            "data"
+            "data",
+            "crypto-asset"
```

Comment:
:information_source: __crypto-asset__ = Cryptographic asset including algorithms, protocols, certificates, keys, tokens, and secrets.